tester/main.cpp

```diff
@@ -301,47 +300,41 @@ int main(int argc, char const *argv[]) {
 	// preparing versions
 	vc::Version::Builder builder;
 	std::list<run_element_t> jobList;
+	vc::vc_utils_init();
```

Comment:
This is part of the high-level APIs of libVC. This example should stick to the low-level ones.